src/DotNet.Testcontainers/Containers/Modules/Databases/CouchbaseTestcontainer.cs

```diff
@@ -0,0 +1,39 @@
+namespace DotNet.Testcontainers.Containers.Modules.Databases
+{
+  using System.Linq;
+  using System.Threading.Tasks;
+  using Abstractions;
+  using Configurations;
+
+  public class CouchbaseTestcontainer : TestcontainerDatabase
+  {
+    internal CouchbaseTestcontainer(ITestcontainersConfiguration configuration) : base(configuration)
+    {
+    }
+
+    public override string ConnectionString => "couchbase://" + this.Hostname;
+
+    public async Task<long> CreateBucket(string bucket, int memory = 512, int withDelaySecond = 3)
```

Comment:
`FlushBucket` and `FlushBucket` not covered by tests. Would be nice to have tests here. Maybe consider xUnit `IAsyncLifetime`, example in `OracleFixture`. I think that's a good way to setup containers for tests.

Reply:
I'll add new tests! Thank you.